blenderproc/python/object/FaceSlicer.py

```diff
@@ -10,6 +10,74 @@
 from blenderproc.python.types.MeshObjectUtility import MeshObject
 from blenderproc.python.utility.Utility import resolve_path
 
+def slice_faces_with_normals(mesh_objects: List[MeshObject], compare_angle_degrees: float = 7.5,
```

Comment:
I think it would make more sense to just pass one object here. If the user wants to apply this function to a list of objects the loop should be outside that function.

Reply:
I agree.